src/components/Filters/Filters.tsx

```diff
@@ -1,91 +1,79 @@
 'use client'
-import styles from './Filters.module.scss'
 import React, { useState } from 'react'
-import 'material-icons/iconfont/material-icons.css'
-import { FilterButton } from './Buttons/FilterButton/FilterButton'
+import styles from './Filters.module.scss'
+import { DropdownFilter } from '../../inputs/Dropdowns/DropdownFilter/DropdownFilter'
 import { DevTypeButton } from './Buttons/DevTypeButton/DevTypeButton'
 
-const Filters = () => {
-  const [selectedTechnology, setSelectedTechnology] = useState<string[]>([])
-  const [selectedSeniority, setSelectedSeniority] = useState<string[]>([])
-  const [selectedAvailability, setSelectedAvailability] = useState<string[]>([])
-  const [selectedLocation, setSelectedLocation] = useState<string[]>([])
-
-  const handleSelect = (option: string, buttonType: string) => {
-    let updatedOptions: string[]
-
-    const updateSelectedOptions = (
-      selectedOptions: string[],
-      setter: (value: string[]) => void,
-    ) => {
-      updatedOptions = selectedOptions.includes(option)
-        ? selectedOptions.filter((selectedOption) => selectedOption !== option)
-        : [...selectedOptions, option]
-      setter(updatedOptions)
-    }
-
-    switch (buttonType) {
-      case 'Technology':
-        updateSelectedOptions(selectedTechnology, setSelectedTechnology)
-        break
-
-      case 'Seniority':
-        updateSelectedOptions(selectedSeniority, setSelectedSeniority)
-        break
-
-      case 'Availability':
-        updateSelectedOptions(selectedAvailability, setSelectedAvailability)
-        break
+const Filters: React.FC = () => {
+  const initialState = {
+    technology: [] as string[],
```

Comment:
**nitpick:** We should move it out from component body, and type it properly (type object, don't do `as` something)